src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font-family to ensure text is displayed with a default font even if the primary font fails to load. For example: `font-family: Roboto, Arial, sans-serif;`.